src/smc-util/db-schema/types.ts

```diff
@@ -49,15 +49,14 @@ interface TableSchema<F extends Fields> {
       instead_of_query?: (
         database,
         obj,
-        instead_of_query,
```

Comment:
I have no idea why there is `instead_of_query` inside of `instead_of_query`

Reply:
Yes, what he had made little sense.  But also that obj should be the typescript version of this (due to my later changes):
```
opts:
            account_id : undefined
            project_id : undefined
            table      : required
            query      : required
            multi      : required
            options    : required
```